package.json

```
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
```

Ideally you should have at least a `build` script here:

```suggestion
  "scripts": {
    "build": "scss src/scss:css"
  },
  "dependencies": {
```